### PR TITLE
Update the BoringSSL instructions to match current paths,

### PR DESCRIPTION
**Pull request by morlovich**
submodule update directions (thanks to Steve to help with that!)

### CODE REVIEW

**Comment by morlovich** [comment]
Jeff, this should re-run .gyp based on the .gypi changing automatically, right?

File: third_party/serf/UPDATING_BORINGSSL [hunk 1]
```diff
@@ -26,6 +40,3 @@ http://googlechromereleases.blogspot.com/search/label/Stable%20updates
     cp -r src/third_party/boringssl/$arch $SERF_DIR/$arch/
   done
```
Comment (hillsp) [inline third_party/serf/UPDATING_BORINGSSL:42]: Dunno if it's worth calling out that you need to cd back into the git directory, commit stuff and push? It's probably also a good idea to spin a build before pushing at least.

But this looks much better, thanks.

**Comment by hillsp** [comment]
On the subject of gyp, I don't think we have any automatic rules that update makefiles with the gypfiles change. One just has to re-run gyp "manually"

**Comment by jeffkaufman** [comment]
@morlovich wrote:
> this should re-run .gyp based on the .gypi changing automatically, right?

@hillsp wrote:
> I don't think we have any automatic rules that update makefiles with the gypfiles change. One just has to re-run gyp "manually"

All existing gyp(i) files are listed in `mod_pagespeed/gyp_manifest.d`, a temporary file that's created when you first start using `devel/Makefile` and loaded as `GYP_FILES`  Then if any of those existing gyp files change that invalidates `HOOKS_STAMP`, making `gyp` rerun itself:

```
$(HOOKS_STAMP) : $(GPERF) $(GYP_FILES)
        bash -o pipefail -c \
          "cd $(GIT_SRC) && python build/gyp_chromium --depth=. | \              
             egrep -v '^(Generating )[a-zA-Z0-9_/.-]*(Makefile|mk)$$'"
        touch $(HOOKS_STAMP)
```

**Review by jeffkaufman — APPROVED** [review]
LGTM

**Pull request merged**